security-proxy/src/deb/resources/etc/georchestra/security-proxy/targets-mapping.properties

```diff
@@ -0,0 +1,11 @@
+analytics=http://localhost:8080/analytics/
+catalogapp=http://localhost:8080/catalogapp/
+downloadform=http://localhost:8080/downloadform/
+extractorapp=http://localhost:8080/extractorapp/
+geonetwork=http://localhost:8080/geonetwork/
+geoserver=http://localhost:8180/geoserver/
+geofence=http://localhost:8180/geofence/
+header=http://localhost:8080/header/
+ldapadmin=http://localhost:8080/ldapadmin/
+mapfishapp=http://localhost:8080/mapfishapp/
+static=http://localhost:8080/header/
```

Comment:
According to the doc, shouldn't the port rather be 8280 for tomcat-georchestra and 8380 for tomcat-geoserver0 ?